data/governments.txt

```diff
@@ -268,36 +268,6 @@ government "Forest (Prey)"
 		destroy 0
 		atrocity 0
 
-government "Forest (Predator)"
```

Comment:
This government is still referred to in the `"Pirate (Devil-Run Gang)"` definition, you'll need to remove that line as well.
https://github.com/endless-sky/endless-sky/blob/a45fb00b7f03f22844b81452546eaecd03fddae5/data/governments.txt#L953